frontend/components/main.js

```diff
@@ -48,11 +48,12 @@ export default class Main extends React.Component {
   }
 
   static propTypes = {
-    match: PropTypes.object.isRequired,
-  }
-
-  get category() {
-    return this.props.match.params.category
+    pageContext: {
```

Comment:
`pageContext` is a Gatsby thing or a React thing?

Reply:
Gatsby thing. Usually it gets piped through a GraphQL query, but GraphQL adds unnecessary complexity to what we're doing so I used this approach instead:

https://www.gatsbyjs.org/docs/using-gatsby-without-graphql/#the-approach-fetch-data-and-use-gatsbys-createpages-api

Reply:
I added a comment about this.